index.js

```diff
@@ -24,13 +24,14 @@ class CacheDigest {
   // Executed when each compilation is finished.
   // Examples: Hot-reload (send a websocket push).
   onCompile(files, publicFiles) {
-    if (this.env == "production") {
-      this.renameFiles(files, false);
-      this.renameFiles(publicFiles, true);
-      this.renameFiles(this.files, true);
+    if (!global.buildingWebComponents) {
```

Comment:
I know we are about the only ones using this but could this have a little bit more global name.
Like processingPrecompile or something like that.